fusor-ember-cli/app/adapters/deployment.js

```diff
@@ -1,5 +1,12 @@
 import ApplicationAdapter from './application';
 
 export default ApplicationAdapter.extend({
-    namespace: 'fusor/api/v21'
+  namespace: 'fusor/api/v21',
+
+  urlForQueryRecord(query, modelName) {
+    if (query.deployment_id) {
+      return `/${this.get('namespace')}/deployments/${query.deployment_id}`;
+    }
```

Comment:
/deployments for index
/deployments/:id for show
is the standard rails and ember convention for routing, so it should work out of the box without overriding `urlForQueryRecord` unless I'm missing something.